src/components/network/line-layer.js

```diff
@@ -428,6 +406,54 @@ class LineLayer extends CompositeLayer {
         this.setState({ compositeData: compositeData });
     }
 
+    recomputeParallelLinesIndex(compositeData, props) {
+        compositeData.forEach((compositeData) => {
+            const mapOriginDestination = compositeData.mapOriginDestination;
+            // calculate index for line with same subsation set
+            // The index is a real number in a normalized unit.
+            // +1 => distanceBetweenLines on side
+            // -1 => distanceBetweenLines on the other side
+            // 0.5 => half of distanceBetweenLines
```

Comment:
Why did you remove the 9999. special case ?

Reply:
we always want a forkline to connect to the correct substation so it is a 0 offset when we do not want parallel line

Reply:
Then we should remove the 9999. handling in the shaders as well.

We had the 9999. special value because we don't want the arrow to stop before reaching the substation in dynamic mode. So we lose this in favor of having the line connect to the correct ring of the voltage level